docs/shape_and_batch_size.md

```diff
@@ -6,6 +6,9 @@ It accepts also a value `auto` - this special phrase make the served model to se
 - Each time the input data change the batch size, the model is reloaded. It might have extra response delay for the first request.
 This feature is useful for sequential inference requests of the same batch size.
 
+*Note:* In case of frequent batch size changes in predict requests consider using [demultiplexing feature](./demultiplexing.md#dynamic-batch-handling-with-demultiplexing) from [Directed Acyclic Graph Scheduler](./dag_scheduler.md) which is more
+performant in such situations.
```

Comment:
is more performant because it is not adding extra overhead on model reloading between requests like with `--batch_size auto` setting.